1-syntax-basic-constructs/basic-js.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <script src="basic.js"></script>
```

Comment:
```suggestion
    <script src="basic.js" defer></script>
```
Usar defer o mueve la etiqueta antes de cerrar el body para no bloquear la descarga del HTML.